src/facade/dragonfly_connection.cc

```diff
@@ -112,6 +112,16 @@ ABSL_FLAG(uint32_t, pipeline_wait_batch_usec, 0,
           "If non-zero, waits for this time for more I/O "
           " events to come for the connection in case there is only one command in the pipeline. ");
 
+ABSL_FLAG(size_t, connection_disk_backpressure_watermark, 0,
```

Comment:
what is the unit here? what crosses the watermark?

Reply:
`dispatch_q_ > watermark` 

Will edit the comment/description accordingly